aws/resource_aws_emr_security_configuration.go

```diff
@@ -26,27 +25,13 @@ func resourceAwsEMRSecurityConfiguration() *schema.Resource {
 				Computed:      true,
 				ForceNew:      true,
 				ConflictsWith: []string{"name_prefix"},
-				ValidateFunc: func(v interface{}, k string) (ws []string, errors []error) {
-					value := v.(string)
-					if len(value) > 10280 {
-						errors = append(errors, fmt.Errorf(
-							"%q cannot be longer than 10280 characters", k))
-					}
-					return
-				},
+				ValidateFunc:  validateMaxLength(10280),
 			},
 			"name_prefix": &schema.Schema{
-				Type:     schema.TypeString,
-				Optional: true,
-				ForceNew: true,
-				ValidateFunc: func(v interface{}, k string) (ws []string, errors []error) {
-					value := v.(string)
-					if len(value) > 10000 {
-						errors = append(errors, fmt.Errorf(
-							"%q cannot be longer than 10000 characters, name is limited to 10280", k))
-					}
-					return
-				},
+				Type:         schema.TypeString,
+				Optional:     true,
+				ForceNew:     true,
+				ValidateFunc: validateMaxLength(10000),
```

Comment:
You mind adding a comment here about the 10280 as the actual limit? And maybe doing it as the subtraction like you have done elsewhere?

Reply:
synced with new const 😏